HANDBOOK.md

```diff
@@ -238,31 +237,31 @@ It is your choice to build the barriers first or the map first. For this example
   ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/97132307-260e-4d1a-b065-f909d30ee39b)
 
 * From the ctf_map README:
-  >`initial_stuff` are the items given to players at their (re)spawn. The `initial_stuff` field is located in the `map.conf` file. At least a pickaxe and some torches should be given in the map's `initial_stuff`.
+  > `initial_stuff` are the items given to players at their (re)spawn. The `initial_stuff` field is located in the `map.conf` file. At least a pickaxe and some torches should be given in the map's `initial_stuff`.
 
-  >An example of `initial_stuff` value that registers a stone pickaxe, 30 cobblestones, 5 torches and a pistol is given below.
+  > Below is an example of `initial_stuff` value that registers a stone pickaxe, 30 cobblestones, 5 torches and a pistol.
```

Comment:
```suggestion
  > An example of `initial_stuff` value that registers a stone pickaxe, 30 cobblestones, 5 torches and a pistol is given below.
```
This is a straight out quote from the ctf_map README, so I'm not editing unless the original text was edited.